specification/logic/resource-manager/Microsoft.Logic/stable/2019-05-01/examples/IntegrationServiceEnvironments_ManagedApis_Put.json

```diff
@@ -4,7 +4,13 @@
     "subscriptionId": "f34b22a3-2202-4fb1-b040-1332bd928c84",
     "resourceGroup": "testResourceGroup",
     "integrationServiceEnvironmentName": "testIntegrationServiceEnvironment",
-    "apiName": "servicebus"
+    "apiName": "servicebus",
+    "integrationServiceEnvironmentManagedApi": {
+      "location": "brazilsouth",
+      "properties": {
+        "deploymentParameters": null
```

Comment:
Thanks for raising this PR. If it's possible, could you supplement the required parameters in request body for the example?

Reply:
Actually, in a simple most used cases, this parameters would be enough to create a managed api in integration service environment